### ISSUE TITLE
Некорректно указана лицензия под которой распространяется проект

### ISSUE THREAD

**Issue by zlocate** [pachca/pachca-desktop#2]
Приветствую. Обратил внимание что в описании репозитория указано что клиент распространяется под MIT лицензией что не совсем соответствует действительности ибо клиент распространяется в только в виде бинарных сборок.
Будет здорово если вы скорректируете описание и файл лицензии в репо

![изображение](https://user-images.githubusercontent.com/43764701/221378524-553da5d5-87fe-48ba-af09-0abc62da2f23.png)

**Comment by DartPower** [comment]
Согласен, из-за наличия данного файла, не понятно по итогу, является ли данный проект FOSS или нет.
Будем надеяться, что это это не проприетарщина и авторы поделятся исходным кодом, хоты бы по данной лицензии.
